### ISSUE TITLE
Web- instead of bt- interface?

### ISSUE THREAD

**Issue by sirtet** [electricidea/Bluetooth-RoverC#1]
I wonder if a Webserver on the m5stick would also work well as a User interface. 
Did u consider other options than bt-serial?
Why did you choose the bt Serial interface?

**Comment by electricidea** [comment]
Yes, a web server can be implemented on the M5Stick without any problems. But then you need a web page that has to be loaded. This is not a problem, but the BT-Serial connection was easier. And I wanted to work with it for a long time.
I have developed code for a web server for this project:
https://www.hackster.io/hague/m5atom-env-mini-data-monitor-026469
